restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -1727,11 +1816,15 @@ private void outboundToPstn(final CreateCall request, final ActorRef sender) thr
 
 
     private void outboundToSip(final CreateCall request, final ActorRef sender) throws ServletParseException {
+        final String uri = (request.getOutboundProxy()!= null && (!request.getOutboundProxy().isEmpty())) ? request.getOutboundProxy() : "";
```

Comment:
@abdulazizali77 I believe this is wrong here. We shouldn't route SIP calls using outbound proxy.
If we want to dial a specific SIP URI then the SIP URI contains the user and host to reach the destination and we must not change the host to outbound proxy.
Again, I might be missing something from the design and the requirements of the feature so please point me to the doc